src/helpers/index.js

```diff
@@ -0,0 +1,14 @@
+import bcrypt from 'bcrypt';
+
+const generateHash = (password) => {
+	const hashPassword = bcrypt.hashSync(password,bcrypt.genSaltSync(10));
+	return hashPassword;
+}
+
+const compareHashedPassword = (hash,password) => {
+		
+	return bcrypt.compareSync(password,hash);
+}
+	
+
+export default {generateHash,compareHashedPassword};
```

Comment:
put the new line.

Reply:
what do you mean?
like this
```suggestion
export default {
 generateHash,
 compareHashedPassword
};
```